Dangerfile

```diff
@@ -40,7 +40,7 @@
 # content changes within the diff
 @S_SECURITY_CONTENT_PATTERN ||= /auth|password|permission|token|secret|security|scope|api_key|apikey|sudo/
 # dont ever match against changes in these files
-@S_SECURITY_EXCLUDE_FILES ||= /test_.*\.py/
+@S_SECURITY_EXCLUDE_FILES ||= /CHANGES|tests/
```

Comment:
Yeah, I've seen it triggered several times from test cases that didn't fit that pattern. I can add `tests` in addition to `tests_.*\.py` if you'd prefer. But I was attempting to also cover JS tests, etc.